sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/SymbolHelper.cs

```diff
@@ -60,6 +60,10 @@ public static IEnumerable<INamedTypeSymbol> GetAllNamedTypes(this INamedTypeSymb
             }
         }
 
+        public static bool IsInherited(this IMethodSymbol symbol) => GetInterfaceMember(symbol) != null;

```

Comment:
I am not a big fan of these helpers:
- doesn't look like the kind of thing we are going to reuse (if I am wrong then we will extract this later on)
- when you implement (override) an abstract member I tend to think this will belong into this `IsInherited` method.